Firebase/Core/FIROptions.m

```diff
@@ -109,6 +110,12 @@ + (FIROptions *)defaultOptions {
 
 #pragma mark - Private class methods
 
++ (void)load {
```

Comment:
I'm not sure if the order of the classes is always that FIRLogger is before Analytics classes as classes might get registered at different order at runtime.